### PR TITLE
Optimizing weh_hub.Start() for master

### PR DESCRIPTION
**Pull request by coreyhulen**

### CODE REVIEW

File: app/web_conn.go [hunk 1]
```diff
@@ -29,6 +29,7 @@ type WebConn struct {
 	Send                      chan model.WebSocketMessage
 	SessionToken              string
 	SessionExpiresAt          int64
+	Session                   *model.Session
```
Comment (crspeller) [inline app/web_conn.go]: You should also set this in the `IsAuthenticated` function.

Comment (coreyhulen) [inline app/web_conn.go]: added.

File: app/web_conn.go [hunk 2]
```diff
@@ -167,6 +169,7 @@ func (webCon *WebConn) IsAuthenticated() bool {
 
 		webCon.SessionToken = session.Token
 		webCon.SessionExpiresAt = session.ExpiresAt
+		webCon.Session = session
```
Comment (crspeller) [inline app/web_conn.go]: You need to clear it in the not found case above as well.

Comment (coreyhulen) [inline app/web_conn.go]: done

**Pull request merged**